src/NHSD.BuyingCatalogue.Identity.Api/Settings/ClientSettingCollection.cs

```diff
@@ -16,6 +16,7 @@ public sealed class ClientSetting
         public string AllowedGrantTypes { get; set; }
         public bool AllowOfflineAccess { get; set; }
         public bool RequireClientSecret { get; set; }
+        public bool RequirePKCE { get; set; }
```

Comment:
I think `RequirePkce` would probably be more in keeping with the .NET casing/naming guidelines.